electron/index.js

```diff
@@ -118,7 +124,7 @@ class CapacitorSplashScreen {
       }
     });
 
-    this.splashWindow.loadURL(`data:text/html;charset=UTF-8,${splashHtml}`, {baseURLForDataURL: `file://${rootPath}/splash_assets/`});
```

Comment:
This is the response to Electron bug: https://github.com/electron/electron/issues/20700